performance-test/scripts/Esignet_Mockida_Test_Script.jmx

```diff
@@ -0,0 +1,1511 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.3">
+  <hashTree>
+    <TestPlan guiclass="TestPlanGui" testclass="TestPlan" testname="Test Plan" enabled="true">
+      <stringProp name="TestPlan.comments"></stringProp>
```

Comment:
User Defined Variables:
1. Why we have acrValuesPwd ? As per the scope it is only otp based auth right.
2. In case of this mock script, we don't have pms in the dependent service, why do we have pmsClientId and other properties ?

Reply:
@vishwa-vyom 
1. We have changed acrValuesPwd to acrValuesOtp specifically pointing to OTP authentication.
2. Client ID generated by eSignet needs auth token generated by PMS client.

Reply:
@nihallakshmankonchady In that case kindly add PMS details in the request sheet as well. As there is a dependency on that.

Reply:
If the eSignet mock setup is enabled with authentication, we need a clientId and secret key to generate the access token. It is not necessary to the PMS client id, it can be any client id with the proper roles.
So, doing this auth token generation is fine, but lets not call that clientId and secret key as pms anywhere, since mock is not directly / indirectly dependent on PMS.

Reply:
We have removed Auth token generation as well as PMS dependency.